### ISSUE TITLE
Partial matching in "plotQualificationComparisonTimeProfile" may result in wrong observed data selection

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/Reporting-Engine#272]
When looking for the mapped observed data set in the list of all data sets:

https://github.com/Open-Systems-Pharmacology/Reporting-Engine/blob/develop/src/code/qualification/plotQualificationComparisonTimeProfile.m#L151

`findPathOutput = contains(Curves.ObservedData,ObservedDataSets(j).Id)`

If there are two datasets, e.g. "myData" and "myData_2", when trying to plot "myData_2", actually "myData" will be plottted.

**Comment by PavelBal** [comment]
`strcmp` instead of `contains` does the trick.

**Comment by PavelBal** [comment]
No one cares? I could push a fix but I am not allowed to create branches.

**Comment by Yuri05** [comment]
create a PR from fork 😉

**Comment by msevestre** [comment]
Yes. This is the standard workflow with GitHub. Create a fork and PR from your fork

**Comment by PavelBal** [comment]
Well, we never had this workflow in R (or core), and I am used to that.
